ocs_ci/ocs/machine.py

```diff
@@ -120,3 +119,79 @@ def delete_machine_and_check_state_of_new_spinned_machine(machine_name):
         log.info(f"{new_machine.name} is in {state} state")
         return state == constants.STATUS_RUNNING.islower()
     return False
+
+
+def get_machinesets():
+    """
+    Get machine sets
+
+    Returns:
+        machine_sets (list): list of machine sets
+    """
+    machine_sets = list()
+    machinesets_obj = OCP(kind=constants.MACHINESETS, namespace=constants.OPENSHIFT_MACHINE_API_NAMESPACE)
```

Comment:
I think it would be better to have OCS objects for machine sets

Reply:
These ops are OCP related